src/build-scripts/build_openexr.bash

```diff
@@ -64,9 +62,8 @@ popd
 
 # Set up paths. These will only affect the caller if this script is
 # run with 'source' rather than in a separate shell.
-export ILMBASE_ROOT=$OPENEXR_INSTALL_DIR
+export OpenEXR_ROOT=$OPENEXR_INSTALL_DIR
 export OPENEXR_ROOT=$OPENEXR_INSTALL_DIR
```

Comment:
For Windows 2022, it's pretty strange but this seems to make it work.
https://github.com/brechtvl/OpenShadingLanguage/actions/runs/12959852395/job/36152802147

```suggestion
+# Environment variables are case insensitive on Windows, don't set
+# it a second time with different case. This seems to trigger a bug
+# where the next cmake command fails to find the compiler.
+if [[ $(uname -s) != "Windows" && "${RUNNER_OS}" != "Windows" ]]; then
+  export OPENEXR_ROOT=$OPENEXR_INSTALL_DIR
+fi
```

Reply:
Holy cow, you are a life saver for this one! I've spent a lot of hours trying to understand what's going on here, and never in a million years would I have suspected this innocent line to be the culprit.

Reply:
I feel like I've been around computers for a very long time now, and yet I don't have even the vaguest guess how setting both OPENEXR_ROOT and OpenEXR_ROOT would start a cascade of problems that leads to the OIIO cmake build not being able to figure out which C++ compiler it should be using.

Reply:
Bash manipulates the `extern char **environ;` array.
https://github.com/bminor/bash/blob/master/variables.c#L4997

I suspect that somewhere in the Bash -> MinGW -> MSVC runtime -> OS stack, it ends up using the wrong index and overwriting an unrelated environment variable, or considers the array invalid due to duplicate entries and discards it.